src/App.css

```diff
@@ -16,7 +16,8 @@ h1 {
 }
 
 ul {
-  margin-top: 40px;
+  margin-top: 0;
+  padding: 0;
```

Comment:
:wrench: Let's use margin and padding from material instead.

Reply:
Addressed:  2c594aaa7d40e5110355ebdfb9a319ce72161bd8